api/v1alpha1/flowcollector_types.go

```diff
@@ -253,6 +253,10 @@ type FlowCollectorFLP struct {
 	//+kubebuilder:default:=true
 	// EnableKubeProbes is a flag to enable or disable Kubernetes liveness/readiness probes
 	EnableKubeProbes bool `json:"enableKubeProbes,omitempty"`
+
+	//+kubebuilder:default:=""
+	// MetricsConfigFile is the configuration file containing specific metrics to be derived
+	MetricsConfigFile string `json:"metricsConfigFile,omitempty"`
```

Comment:
we do not need to expose any of this to the users, just embed the .yaml file in the code like what @ronensc suggested

Reply:
OK